packages/shadydom/src/utils.js

```diff
@@ -252,13 +252,21 @@ export const patchExistingProperties = (proto, descriptors) => {
  * and the property will be patched with a valued function (not accessor), then
  * copy the existing property value to the native location for the value.
  */
-export const copyImplementedNativeProperties = (proto, descriptors) => {
+export const copyImplementedNativeProperties = (
+  proto,
+  descriptors,
+  disallowed
+) => {
   for (let name in descriptors) {
     const d = descriptors[name];
     // Only check methods.
```

Comment:
Why not also give this treatment for accessors? Might be nice to not need to come in and add it later if we decide we need a className override or something...? 🤷